setup.py

```diff
@@ -35,6 +35,9 @@
         'console_scripts': [
             'dallinger = dallinger.command_line:dallinger',
         ],
+        'dallinger.experiments': [
+            'Bartlett1932 = demos.bartlett1932.experiment:Bartlett1932',
```

Comment:
Yes, and any 3rd party experiment can provide a setup.py that registers an experiment class in the same manner.